recipes/moc/all/conanfile.py

```diff
@@ -0,0 +1,149 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from os import rename
+from os.path import isdir, join
+from shutil import move
+from glob import glob
+from distutils.version import LooseVersion
+
+class MocConan(ConanFile):
+    _noCmakeConfigFiles = True
+    _requirements = [
+        "flex/2.6.4",
+        "bison/3.7.1"
+    ]
```

Comment:
```suggestion
```

Reply:
so moc build depends upon these . I'm not certain what you're suggesting.

Reply:
I think @intelligide suggests to avoid such indirection, recipe is harder to understand. Please try to follow style of others recipes in CCI